WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -53,7 +53,7 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 	{
 		var coinChanges =
 			_wallet.Coins.List.ToObservableChangeSet(model => model.Key)
-				.TransformWithInlineUpdate(x => new WalletCoinViewModel(x))
+				.TransformWithInlineUpdate(x => new WalletCoinViewModel(x), (x, y) => { }) // TODO: SuperJMN: is this safe?
```

Comment:
@SuperJMN

Reply:
Yes, this is safe in this scenario. Before, we have an old version of this method, that didn't provide any update action. Now it's mandatory, but we're not updating the ViewModel when a coin is updated. In the future this could change, but it's fine for now.